src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/Example1.java

```diff
@@ -0,0 +1,58 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="Indentation"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;
+
+// xdoc section -- start
+class Example1 {
+    String field = "example";                  // basicOffset
+    int[] values = {                           // basicOffset
+        10,
+        20,
+        30
+    };
+
+    void processValues() throws Exception {
+        handleValue("Test String", 42);          // basicOffset
+    }
+
+    void handleValue(String aFooString,
+                     int aFooInt) {             // indent:8 ; expected: > 4;
+        boolean cond1 = true;
+        boolean cond2 = false;
+        boolean cond3 = true;
+        boolean cond4 = false;
+        boolean cond5 = true;
+        boolean cond6 = false;
```

Comment:
Can we collapse this block of declarations to single statement, no need to define default values.
It should fit in one line.

So we might not need suppression on file size anymore

Reply:
i have removed the supression .

Reply:
there was no need for the supression.

Reply:
Should i leave the variables uninitialized??
if i leave them uninitialized it throws error

Reply:
Can you try this https://stackoverflow.com/a/6202907/1015848

Reply:
done